lib/Bio/EnsEMBL/Tark/Schema/Result/Assembly.pm

```diff
@@ -285,6 +285,15 @@ __PACKAGE__->has_many(
 # Created by DBIx::Class::Schema::Loader v0.07045 @ 2019-01-22 14:34:33
 # DO NOT MODIFY THIS OR ANYTHING ABOVE! md5sum:BojvqkqVSY6EaarsUUKuuw
 
+=head2 sqlt_deploy_hook
+  Arg [1]    : $qlt_table : Bio::EnsEMBL::Tark::Schema::Result::Session
```

Comment:
Though it's minor..noticed a discrepancy in spelling "Arg [1]    : **$qlt_table** " and the actual argument "**$sqlt_table**" in this class and all the classes below!!